changelogs/fragments/697-iam_role-replace-UpdateRoleDescription-with-UpdateRole.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- iam_role - Modified iam_role to replace update_role_description with update_role (https://github.com/ansible-collections/community.aws/pull/697).
```

Comment:
Nit, I would call out that this is an internal code change.  Users may not be familiar with the boto functions that we use so they might assume this could affect them.  Something like, 'Modified iam_role internal code to replace...'/